<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.32">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>6&nbsp; Treatment and Control Strategies – *Clostridoides difficile* Profiling</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./chapter_6.html" rel="next">
<link href="./chapter_4.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-2486e1f0a3ee9ee1fc393803a1361cdb.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-69a30c073420f475066ff0fd08b8a642.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./chapter_5.html"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Treatment and Control Strategies</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./"><em>Clostridoides difficile</em> Profiling</a> 
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./preface.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Preface/Disclaimer</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Pathogen Overview</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Virulence mechanisms, environmental persistence and survival</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_3.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Therapeutic Strategies Targeting Colonization Resistance</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_4.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Diagnostic Techniques and Challenges</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_5.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Treatment and Control Strategies</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_6.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Host immune response</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_7.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Future Research Directions and Opportunities</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./reference.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">References</span></span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul class="collapse">
  <li><a href="#antibiotic-therapies" id="toc-antibiotic-therapies" class="nav-link active" data-scroll-target="#antibiotic-therapies"><span class="header-section-number">6.1</span> Antibiotic Therapies</a></li>
  <li><a href="#microbiota-restoration-therapies" id="toc-microbiota-restoration-therapies" class="nav-link" data-scroll-target="#microbiota-restoration-therapies"><span class="header-section-number">6.2</span> Microbiota Restoration Therapies</a></li>
  <li><a href="#toxin-targeting-treatments" id="toc-toxin-targeting-treatments" class="nav-link" data-scroll-target="#toxin-targeting-treatments"><span class="header-section-number">6.3</span> Toxin-Targeting Treatments</a></li>
  <li><a href="#antimicrobial-peptides-and-small-molecule-inhibitors" id="toc-antimicrobial-peptides-and-small-molecule-inhibitors" class="nav-link" data-scroll-target="#antimicrobial-peptides-and-small-molecule-inhibitors"><span class="header-section-number">6.4</span> Antimicrobial Peptides and Small Molecule Inhibitors</a></li>
  <li><a href="#bile-acid-modulation-and-nutrient-limitation" id="toc-bile-acid-modulation-and-nutrient-limitation" class="nav-link" data-scroll-target="#bile-acid-modulation-and-nutrient-limitation"><span class="header-section-number">6.5</span> Bile Acid Modulation and Nutrient Limitation</a></li>
  <li><a href="#emerging-and-experimental-therapies" id="toc-emerging-and-experimental-therapies" class="nav-link" data-scroll-target="#emerging-and-experimental-therapies"><span class="header-section-number">6.6</span> Emerging and Experimental Therapies</a></li>
  <li><a href="#infection-control-measures-in-healthcare-settings" id="toc-infection-control-measures-in-healthcare-settings" class="nav-link" data-scroll-target="#infection-control-measures-in-healthcare-settings"><span class="header-section-number">6.7</span> Infection Control Measures in Healthcare Settings</a></li>
  <li><a href="#infection-control-measures-in-community-settings" id="toc-infection-control-measures-in-community-settings" class="nav-link" data-scroll-target="#infection-control-measures-in-community-settings"><span class="header-section-number">6.8</span> Infection Control Measures in Community Settings</a></li>
  <li><a href="#novel-simulator-for-tracking-hai" id="toc-novel-simulator-for-tracking-hai" class="nav-link" data-scroll-target="#novel-simulator-for-tracking-hai"><span class="header-section-number">6.10</span> Novel simulator for tracking HAI</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Treatment and Control Strategies</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p>Current treatment methods include antibiotics (e.g., vancomycin, fidaxomicin), faecal microbiota transplantation (FMT), and emerging therapies that modulate the gut microbiota to restore balance.</p>
<p>Biocide applications targeting environmental control are also crucial. Opportunities lie in developing formulations that interfere with spore germination, biofilm formation, and nutrient acquisition, all of which are key survival mechanisms for <em>C. difficile.</em></p>
<p><strong>Relevant treatment</strong> <span class="citation" data-cites="sholeh2020">(<a href="reference.html#ref-sholeh2020" role="doc-biblioref">Sholeh et al. 2020</a>)</span></p>
<table class="caption-top table">
<thead>
<tr class="header">
<th>Antibiotic</th>
<th>Use</th>
<th>Resistance</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Vancomycin</td>
<td>First-line treatment for CDI, administered orally</td>
<td>Resistance is rare</td>
<td><a href="https://aricjournal.biomedcentral.com/articles/10.1186/s13756-020-00815-5">ARIC Journal</a></td>
</tr>
<tr class="even">
<td>Fidaxomicin</td>
<td>First-line treatment, especially if vancomycin is ineffective</td>
<td>Resistance is uncommon</td>
<td><a href="https://aricjournal.biomedcentral.com/articles/10.1186/s13756-020-00815-5">ARIC Journal</a></td>
</tr>
<tr class="odd">
<td>Metronidazole</td>
<td>Reserved for mild CDI cases or when other treatments are unsuitable</td>
<td>Rare resistance cases reported</td>
<td><a href="https://aricjournal.biomedcentral.com/articles/10.1186/s13756-020-00815-5">ARIC Journal</a></td>
</tr>
<tr class="even">
<td>Rifaximin</td>
<td>Follow-up treatment after vancomycin to prevent recurrence</td>
<td>Resistance observed</td>
<td><a href="https://aricjournal.biomedcentral.com/articles/10.1186/s13756-020-00815-5">ARIC Journal</a></td>
</tr>
<tr class="odd">
<td>Tigecycline</td>
<td>Alternative treatment, especially in severe cases</td>
<td>Resistance is rare</td>
<td><a href="https://aricjournal.biomedcentral.com/articles/10.1186/s13756-020-00815-5">ARIC Journal</a></td>
</tr>
<tr class="even">
<td>Bezlotoxumab*</td>
<td>Monoclonal antibody used with antibiotics to reduce recurrence</td>
<td>Not applicable (not an antibiotic; no resistance issues)</td>
<td><span class="citation" data-cites="johnson2021">(<a href="reference.html#ref-johnson2021" role="doc-biblioref">Johnson et al. 2021</a>)</span></td>
</tr>
<tr class="odd">
<td>Clindamycin</td>
<td>Not used for CDI; associated with high CDI risk due to resistance</td>
<td>High resistance in <em>C. difficile</em> strains</td>
<td><a href="https://aricjournal.biomedcentral.com/articles/10.1186/s13756-020-00815-5">ARIC Journal</a></td>
</tr>
<tr class="even">
<td>Fluoroquinolones</td>
<td>Not used for CDI; linked to hypervirulent strains with resistance</td>
<td>High resistance; associated with CDI outbreak strains</td>
<td><a href="https://aricjournal.biomedcentral.com/articles/10.1186/s13756-020-00815-5">ARIC Journal</a></td>
</tr>
<tr class="odd">
<td>Cephalosporins</td>
<td>Not used for CDI treatment; often associated with CDI development</td>
<td>High risk of CDI due to gut microbiota disruption. <em>C. difficile</em> strains can show resistance.</td>
<td><a href="https://aricjournal.biomedcentral.com/articles/10.1186/s13756-020-00815-5">ARIC Journal</a></td>
</tr>
<tr class="even">
<td>Penicillins</td>
<td>Not used for CDI treatment; certain types associated with CDI risk</td>
<td>Broad-spectrum penicillins (e.g., ampicillin) may increase CDI risk by altering gut microbiota. Resistance patterns vary.</td>
<td><a href="https://aricjournal.biomedcentral.com/articles/10.1186/s13756-020-00815-5">ARIC Journal</a></td>
</tr>
</tbody>
</table>
<section id="antibiotic-therapies" class="level2" data-number="6.1">
<h2 data-number="6.1" class="anchored" data-anchor-id="antibiotic-therapies"><span class="header-section-number">6.1</span> Antibiotic Therapies</h2>
<section id="vancomycin" class="level3" data-number="6.1.1">
<h3 data-number="6.1.1" class="anchored" data-anchor-id="vancomycin"><span class="header-section-number">6.1.1</span> Vancomycin</h3>
<ul>
<li>An effective oral antibiotic for CDI, often used for moderate to severe cases. It works by inhibiting bacterial cell wall synthesis, helping to reduce <em>C. difficile</em> numbers in the gut.</li>
</ul>
</section>
<section id="fidaxomicin" class="level3" data-number="6.1.2">
<h3 data-number="6.1.2" class="anchored" data-anchor-id="fidaxomicin"><span class="header-section-number">6.1.2</span> Fidaxomicin</h3>
<ul>
<li>A narrow-spectrum antibiotic that targets <em>C. difficile</em> specifically, preserving other gut microbiota and reducing recurrence rates. Often used for initial treatment and recurrent cases.</li>
</ul>
</section>
<section id="metronidazole" class="level3" data-number="6.1.3">
<h3 data-number="6.1.3" class="anchored" data-anchor-id="metronidazole"><span class="header-section-number">6.1.3</span> Metronidazole</h3>
<ul>
<li>Traditionally used for mild to moderate CDI but now less favoured due to lower efficacy compared to other antibiotics.</li>
</ul>
</section>
<section id="rifaximin" class="level3" data-number="6.1.4">
<h3 data-number="6.1.4" class="anchored" data-anchor-id="rifaximin"><span class="header-section-number">6.1.4</span> <strong>Rifaximin</strong></h3>
<ul>
<li>An oral antibiotic that is sometimes used as a follow-up after vancomycin therapy to help prevent recurrence. Rifaximin has limited systemic absorption, primarily acting in the gut, where it targets residual <em>C. difficile</em> cells. It is often reserved for patients who have experienced recurrent CDI.</li>
</ul>
</section>
<section id="tigecycline" class="level3" data-number="6.1.5">
<h3 data-number="6.1.5" class="anchored" data-anchor-id="tigecycline"><span class="header-section-number">6.1.5</span> <strong>Tigecycline</strong></h3>
<ul>
<li>An intravenous antibiotic for severe or refractory CDI cases, especially when other treatment options have been ineffective. Tigecycline works by inhibiting protein synthesis in bacteria, but its use is generally limited to cases where first-line therapies like vancomycin and fidaxomicin are inadequate.</li>
</ul>
<blockquote class="blockquote">
<h3 id="bezlotoxumab-monoclonal-antibody" data-number="6.1.6" class="anchored"><span class="header-section-number">6.1.6</span> Bezlotoxumab (Monoclonal Antibody)</h3>
<ul>
<li>While not an antibiotic, this monoclonal antibody targets toxin B produced by <em>C. difficile</em>, reducing recurrence risk when used alongside antibiotics.</li>
</ul>
<h3 id="clindamycin" data-number="6.1.7" class="anchored"><span class="header-section-number">6.1.7</span> <strong>Clindamycin</strong></h3>
<ul>
<li>While not used for treating CDI, clindamycin is noteworthy because of its association with an increased risk of developing CDI due to its significant impact on gut microbiota. The widespread use of clindamycin has been linked to the selection of <em>C. difficile</em> strains resistant to standard antibiotics.</li>
</ul>
<h3 id="fluoroquinolones-e.g.-ciprofloxacin-levofloxacin" data-number="6.1.8" class="anchored"><span class="header-section-number">6.1.8</span> <strong>Fluoroquinolones (e.g., Ciprofloxacin, Levofloxacin)</strong></h3>
<ul>
<li>Similar to clindamycin, fluoroquinolones are not used to treat CDI but are known to increase CDI risk. Their use is linked to the emergence of hypervirulent <em>C. difficile</em> strains, particularly those resistant to fluoroquinolones. These antibiotics disrupt the gut microbiota significantly, favouring <em>C. difficile</em> overgrowth.</li>
</ul>
<h3 id="cephalosporins" data-number="6.1.9" class="anchored"><span class="header-section-number">6.1.9</span> <strong>Cephalosporins</strong></h3>
<ul>
<li>Not used for CDI treatment but are associated with a high risk of CDI development due to their broad-spectrum activity, which disrupts gut flora. Cephalosporin use is often linked to CDI outbreaks, especially in hospital settings.</li>
</ul>
<h3 id="penicillins-e.g.-ampicillin" data-number="6.1.10" class="anchored"><span class="header-section-number">6.1.10</span> <strong>Penicillins (e.g., Ampicillin)</strong></h3>
<p>Broad-spectrum penicillins like ampicillin are associated with increased CDI risk by disrupting gut microbiota balance. Narrow-spectrum penicillins pose a lower risk but are still considered cautiously in patients with a history of CDI.</p>
</blockquote>
</section>
</section>
<section id="microbiota-restoration-therapies" class="level2" data-number="6.2">
<h2 data-number="6.2" class="anchored" data-anchor-id="microbiota-restoration-therapies"><span class="header-section-number">6.2</span> Microbiota Restoration Therapies</h2>
<section id="fecal-microbiota-transplantation-fmt" class="level3" data-number="6.2.1">
<h3 data-number="6.2.1" class="anchored" data-anchor-id="fecal-microbiota-transplantation-fmt"><span class="header-section-number">6.2.1</span> Fecal Microbiota Transplantation (FMT)</h3>
<ul>
<li>FMT involves transplanting stool from a healthy donor into the patient’s colon to restore gut microbiota diversity, which helps reestablish colonization resistance against <em>C. difficile</em>. This is particularly effective for recurrent CDI.</li>
</ul>
</section>
<section id="probiotics" class="level3" data-number="6.2.2">
<h3 data-number="6.2.2" class="anchored" data-anchor-id="probiotics"><span class="header-section-number">6.2.2</span> Probiotics</h3>
<ul>
<li>Certain probiotics containing beneficial bacterial strains like <em>Lactobacillus</em> and <em>Saccharomyces boulardii</em> may aid in rebalancing gut flora and preventing recurrence, though evidence is mixed on their efficacy in CDI treatment.</li>
</ul>
</section>
<section id="microbiota-based-therapeutic" class="level3" data-number="6.2.3">
<h3 data-number="6.2.3" class="anchored" data-anchor-id="microbiota-based-therapeutic"><span class="header-section-number">6.2.3</span> Microbiota-Based Therapeutic</h3>
<ul>
<li>Emerging therapies include standardized bacterial consortia or synthetic microbiome capsules (e.g., SER-109), which provide select beneficial gut bacteria to restore a balanced microbiota without the need for donor material.</li>
</ul>
</section>
</section>
<section id="toxin-targeting-treatments" class="level2" data-number="6.3">
<h2 data-number="6.3" class="anchored" data-anchor-id="toxin-targeting-treatments"><span class="header-section-number">6.3</span> Toxin-Targeting Treatments</h2>
<section id="monoclonal-antibodies-e.g.-bezlotoxumab" class="level3" data-number="6.3.1">
<h3 data-number="6.3.1" class="anchored" data-anchor-id="monoclonal-antibodies-e.g.-bezlotoxumab"><span class="header-section-number">6.3.1</span> Monoclonal Antibodies (e.g., Bezlotoxumab)</h3>
<ul>
<li>Administered alongside antibiotics, bezlotoxumab targets toxin B, a major virulence factor, to prevent recurrent infections.</li>
</ul>
</section>
<section id="vaccines" class="level3" data-number="6.3.2">
<h3 data-number="6.3.2" class="anchored" data-anchor-id="vaccines"><span class="header-section-number">6.3.2</span> Vaccines</h3>
<ul>
<li>Several vaccines targeting <em>C. difficile</em> toxins (TcdA and TcdB) are under development, aiming to induce immunity that neutralizes the toxins and prevents infection.</li>
</ul>
</section>
<section id="antitoxins" class="level3" data-number="6.3.3">
<h3 data-number="6.3.3" class="anchored" data-anchor-id="antitoxins"><span class="header-section-number">6.3.3</span> Antitoxins</h3>
<ul>
<li>Although experimental, antitoxin approaches aim to neutralize the effects of TcdA and TcdB in active infections, potentially reducing symptoms and severity.</li>
</ul>
</section>
</section>
<section id="antimicrobial-peptides-and-small-molecule-inhibitors" class="level2" data-number="6.4">
<h2 data-number="6.4" class="anchored" data-anchor-id="antimicrobial-peptides-and-small-molecule-inhibitors"><span class="header-section-number">6.4</span> Antimicrobial Peptides and Small Molecule Inhibitors</h2>
<section id="antimicrobial-peptides" class="level3" data-number="6.4.1">
<h3 data-number="6.4.1" class="anchored" data-anchor-id="antimicrobial-peptides"><span class="header-section-number">6.4.1</span> Antimicrobial Peptides</h3>
<ul>
<li>Synthetic or host-derived peptides can disrupt <em>C. difficile</em> cell membranes, providing a targeted approach that minimizes impacts on the gut microbiota.</li>
</ul>
</section>
<section id="small-molecule-inhibitors" class="level3" data-number="6.4.2">
<h3 data-number="6.4.2" class="anchored" data-anchor-id="small-molecule-inhibitors"><span class="header-section-number">6.4.2</span> Small Molecule Inhibitors</h3>
<ul>
<li>Some research focuses on developing small molecules that inhibit <em>C. difficile</em> toxin production or biofilm formation, targeting bacterial processes without killing other gut bacteria.</li>
</ul>
</section>
</section>
<section id="bile-acid-modulation-and-nutrient-limitation" class="level2" data-number="6.5">
<h2 data-number="6.5" class="anchored" data-anchor-id="bile-acid-modulation-and-nutrient-limitation"><span class="header-section-number">6.5</span> Bile Acid Modulation and Nutrient Limitation</h2>
<section id="bile-acid-modifiers" class="level3" data-number="6.5.1">
<h3 data-number="6.5.1" class="anchored" data-anchor-id="bile-acid-modifiers"><span class="header-section-number">6.5.1</span> Bile Acid Modifiers</h3>
<ul>
<li>Treatments that increase secondary bile acid production or inhibit primary bile acids are being studied to reduce spore germination and vegetative cell growth in the gut.</li>
</ul>
</section>
<section id="nutrient-competition" class="level3" data-number="6.5.2">
<h3 data-number="6.5.2" class="anchored" data-anchor-id="nutrient-competition"><span class="header-section-number">6.5.2</span> Nutrient Competition</h3>
<ul>
<li>Limiting access to specific nutrients that <em>C. difficile</em> relies on, such as trehalose, iron, and zinc, may create an unfavourable environment for bacterial colonization and persistence.</li>
</ul>
</section>
</section>
<section id="emerging-and-experimental-therapies" class="level2" data-number="6.6">
<h2 data-number="6.6" class="anchored" data-anchor-id="emerging-and-experimental-therapies"><span class="header-section-number">6.6</span> Emerging and Experimental Therapies</h2>
<section id="bacteriophage-therapy" class="level3" data-number="6.6.1">
<h3 data-number="6.6.1" class="anchored" data-anchor-id="bacteriophage-therapy"><span class="header-section-number">6.6.1</span> Bacteriophage Therapy</h3>
<ul>
<li>Phages that specifically target <em>C. difficile</em> are being investigated as a means to reduce bacterial load without affecting other gut bacteria.</li>
</ul>
</section>
<section id="crispr-cas-systems" class="level3" data-number="6.6.2">
<h3 data-number="6.6.2" class="anchored" data-anchor-id="crispr-cas-systems"><span class="header-section-number">6.6.2</span> CRISPR-Cas Systems</h3>
<ul>
<li>Gene-editing approaches using CRISPR-Cas systems are under exploration to target and deactivate <em>C. difficile</em> genes responsible for virulence or survival in the gut.</li>
</ul>
</section>
<section id="short-chain-fatty-acids-scfas" class="level3" data-number="6.6.3">
<h3 data-number="6.6.3" class="anchored" data-anchor-id="short-chain-fatty-acids-scfas"><span class="header-section-number">6.6.3</span> Short-Chain Fatty Acids (SCFAs)</h3>
<ul>
<li>SCFAs, produced by beneficial gut bacteria, lower gut pH and inhibit <em>C. difficile</em> growth. Supplementing SCFAs or promoting SCFA-producing bacteria may help maintain colonization resistance.</li>
</ul>
</section>
</section>
<section id="infection-control-measures-in-healthcare-settings" class="level2" data-number="6.7">
<h2 data-number="6.7" class="anchored" data-anchor-id="infection-control-measures-in-healthcare-settings"><span class="header-section-number">6.7</span> Infection Control Measures in Healthcare Settings</h2>
<section id="rigorous-hygiene-and-cleaning" class="level3" data-number="6.7.1">
<h3 data-number="6.7.1" class="anchored" data-anchor-id="rigorous-hygiene-and-cleaning"><span class="header-section-number">6.7.1</span> Rigorous Hygiene and Cleaning</h3>
<ul>
<li>Use of sporicidal agents (e.g., bleach-based cleaners) on surfaces in healthcare facilities to eliminate <em>C. difficile</em> spores and prevent transmission.</li>
</ul>
</section>
<section id="hand-hygiene" class="level3" data-number="6.7.2">
<h3 data-number="6.7.2" class="anchored" data-anchor-id="hand-hygiene"><span class="header-section-number">6.7.2</span> Hand Hygiene</h3>
<ul>
<li>Handwashing with soap and water is recommended over alcohol-based sanitizers, as alcohol is less effective at killing <em>C. difficile</em> spores.</li>
</ul>
</section>
<section id="isolation-precautions" class="level3" data-number="6.7.3">
<h3 data-number="6.7.3" class="anchored" data-anchor-id="isolation-precautions"><span class="header-section-number">6.7.3</span> Isolation Precautions</h3>
<ul>
<li>Infected patients are often isolated, and healthcare workers use protective equipment (e.g., gloves, PPE) to prevent cross-contamination.</li>
</ul>
</section>
<section id="antibiotic-stewardship" class="level3" data-number="6.7.4">
<h3 data-number="6.7.4" class="anchored" data-anchor-id="antibiotic-stewardship"><span class="header-section-number">6.7.4</span> Antibiotic Stewardship</h3>
<ul>
<li>Reducing unnecessary antibiotic use to prevent disruption of gut microbiota and lower the risk of CDI, particularly in hospitals and long-term care facilities.</li>
</ul>
</section>
</section>
<section id="infection-control-measures-in-community-settings" class="level2" data-number="6.8">
<h2 data-number="6.8" class="anchored" data-anchor-id="infection-control-measures-in-community-settings"><span class="header-section-number">6.8</span> Infection Control Measures in Community Settings</h2>
<ul>
<li><p>Strategies include monitoring <em>C. difficile</em> in livestock and regulating antibiotic use in agriculture</p></li>
<li><p>Enhanced Surveillance in Travel and Healthcare Settings: Cross-border ribotype tracking, especially in regions with high patient mobility, has become a priority to prevent international spread of hypervirulent ribotypes.</p></li>
</ul>
<blockquote class="blockquote">
<h2 id="modelling-control" data-number="6.9" class="anchored"><span class="header-section-number">6.9</span> Modelling control</h2>
<p>Stochastic models, specifically MCMC and Bayesian inference, enable accurate CDI tracking by accommodating random infection spread, handling data gaps, and analyzing population-wide immunity. These methods provide insight into how infection risks change and how interventions might affect overall outcomes, essential for devising targeted infection control strategies in hospital settings</p>
<ul>
<li><p>Stochastic Models handle randomness, making them ideal for unpredictable events like concert arrivals or sporadic CDI cases.</p></li>
<li><p>Monte Carlo Simulations explore numerous possible scenarios by running repeated trials, providing a range of outcomes and highlighting patterns.</p></li>
<li><p>Bayesian Statistics incorporate both observed and missing data to continually refine predictions, helping address uncertainties in real-life and medical contexts.</p></li>
</ul>
<p>By combining these approaches, epidemiologists and event planners alike can make better-informed decisions, accounting for randomness, refining their models with each new data point, and exploring a wide range of possible scenarios.</p>
</blockquote>
<section id="spatio-temporal-time-series-models" class="level3" data-number="6.9.1">
<h3 data-number="6.9.1" class="anchored" data-anchor-id="spatio-temporal-time-series-models"><span class="header-section-number">6.9.1</span> Spatio-temporal Time Series Models:</h3>
<p>Spatio-temporal models are mathematical tools used to understand how CDI infections spread over time and across different locations, which is crucial for developing effective infection control strategies.</p>
<ul>
<li><p>These methods, like STARIMA (Spatiotemporal Autoregressive Integrated Moving Average), rely on autocorrelation, meaning they look at how sequential data points (e.g., infection cases over time) are related. They create rules to describe trends over time without assuming specific biological mechanisms of disease spread.</p></li>
<li><p>Advantages: These models are good for examining long-term trends, especially in large datasets where values change smoothly, like annual counts of <em>C. difficile</em> cases across a country.</p></li>
<li><p>Limitations: These models smooth out random fluctuations and struggle with “spiky” data (e.g., sudden, sporadic outbreaks). They are also sensitive to missing data, which can weaken their predictions.</p></li>
</ul>
</section>
<section id="deterministic-models" class="level3" data-number="6.9.2">
<h3 data-number="6.9.2" class="anchored" data-anchor-id="deterministic-models"><span class="header-section-number">6.9.2</span> Deterministic Models:</h3>
<ul>
<li><p>These models work by defining transition rates between disease states (e.g., susceptible to infected, infected to resistant). This approach is based on the biological understanding of infection dynamics and can be applied widely in epidemiology.</p></li>
<li><p>Advantages: Deterministic models allow researchers to include assumptions about how infections progress. They work well with complete data and allow for testing different scenarios, helping evaluate the impact of risk factors like patient proximity.</p></li>
<li><p>Limitations: Similar to time series models, deterministic approaches smooth out data. This is fine for analyzing large datasets (e.g., national case numbers), but not useful for small datasets where slight fluctuations can impact the results (e.g., cases in a single hospital ward).</p></li>
</ul>
</section>
<section id="stochastic-models" class="level3" data-number="6.9.3">
<h3 data-number="6.9.3" class="anchored" data-anchor-id="stochastic-models"><span class="header-section-number">6.9.3</span> Stochastic Models:</h3>
<ul>
<li>Newer mathematical methods can handle stochastic events, or random chance variations. These are particularly useful in cases with small numbers where minor fluctuations (like one or two extra cases) are significant, such as tracking real-time cases in a single hospital unit.</li>
</ul>
<p>Stochastic models, especially Markov chain Monte Carlo (MCMC) methods, are valuable for understanding the spread of <em>Clostridioides difficile</em> (C. difficile) infections in hospital settings because they incorporate randomness, essential for accurately modelling outbreaks that are irregular in time and space. Here’s how these concepts relate and can be organized logically:</p>
<section id="why-stochastic-models-for-cdi" class="level4" data-number="6.9.3.1">
<h4 data-number="6.9.3.1" class="anchored" data-anchor-id="why-stochastic-models-for-cdi"><span class="header-section-number">6.9.3.1</span> 1. Why Stochastic Models for CDI?</h4>
<ul>
<li><p>CDI outbreaks often occur in clusters (short bursts in specific locations), with varying outbreak sizes. For example, outbreaks can range widely, from a few cases to hundreds.</p></li>
<li><p>Unlike deterministic models, which smooth out data and are suitable for larger datasets (like annual national cases), stochastic models are better at capturing the random fluctuations of sporadic outbreaks. These models incorporate chance events, making them useful when patient cases are limited or unpredictable.</p></li>
</ul>
</section>
<section id="using-markov-chains-to-model-cdi" class="level4" data-number="6.9.3.2">
<h4 data-number="6.9.3.2" class="anchored" data-anchor-id="using-markov-chains-to-model-cdi"><span class="header-section-number">6.9.3.2</span> 2. Using Markov Chains to Model CDI</h4>
<ul>
<li><p>Markov Chains are mathematical sequences that model how an event (like infection spread) moves from one state to another (e.g., susceptible to infected). Originally used in atomic physics, Markov chains also apply to infections, as they can simulate random “chain reactions” similar to how infectious particles spread among susceptible individuals.</p></li>
<li><p>By implementing continuous-time data (where events are tracked even if observation intervals vary), Markov models track infections realistically, modeling the disease’s rapid spread and clustering over time.</p></li>
</ul>
</section>
<section id="monte-carlo-simulations-and-bayesian-inference" class="level4" data-number="6.9.3.3">
<h4 data-number="6.9.3.3" class="anchored" data-anchor-id="monte-carlo-simulations-and-bayesian-inference"><span class="header-section-number">6.9.3.3</span> 3. Monte Carlo Simulations and Bayesian Inference</h4>
<ul>
<li><p>Monte Carlo Simulations allow multiple simulations of a Markov chain. This approach enables modelers to examine numerous potential outcomes based on the same infection spread model.</p></li>
<li><p>Bayesian inference further supports these models by estimating missing information and hidden states, which is essential in infection tracking. Bayesian methods in CDI modeling can:</p>
<ol type="1">
<li><p>Infer events between observation times, filling in the gaps.</p></li>
<li><p>Estimate the effects of missing data, accounting for unobserved cases.</p></li>
<li><p>Identify “hidden” infection states (e.g., whether certain individuals are resistant or susceptible).</p></li>
</ol></li>
</ul>
</section>
<section id="application-to-population-immunity" class="level4" data-number="6.9.3.4">
<h4 data-number="6.9.3.4" class="anchored" data-anchor-id="application-to-population-immunity"><span class="header-section-number">6.9.3.4</span> 4. Application to Population Immunity</h4>
<ul>
<li><p>Stochastic models consider the “herd immunity” effect, assessing infection risk across the whole population rather than isolated individuals. The population’s overall susceptibility or resistance influences infection dynamics, which is essential for evaluating preventative measures.</p></li>
<li><p>When the population is close to a critical susceptibility threshold (where slight changes drastically impact infection rates), having complete data is crucial. Bayesian methods help manage this by reducing missing data, allowing for precise model parameters.</p></li>
</ul>
</section>
</section>
</section>
<section id="novel-simulator-for-tracking-hai" class="level2" data-number="6.10">
<h2 data-number="6.10" class="anchored" data-anchor-id="novel-simulator-for-tracking-hai"><span class="header-section-number">6.10</span> Novel simulator for tracking HAI</h2>
<p>A realistic approach that includes patient interactions, hospital layout, and infection control policies. Traditional models for studying infection spread often use compartmental epidemiological models or contact networks, which are useful but lack essential hospital-specific details, such as patient location tracking, shift patterns, and spatial configurations. By integrating these overlooked aspects, this simulator provides a more practical and tailored model for hospital settings.</p>
<section id="key-features-of-the-simulator" class="level3" data-number="6.10.1">
<h3 data-number="6.10.1" class="anchored" data-anchor-id="key-features-of-the-simulator"><span class="header-section-number">6.10.1</span> Key Features of the Simulator:</h3>
<ul>
<li><p>Agent-Based Patient Modeling: Each patient is represented as an agent with individual characteristics, creating a more personalized and dynamic simulation of disease spread.</p></li>
<li><p>Spatial-Temporal Hospital Constraints: The model incorporates physical hospital layout and movement constraints, making it possible to simulate infection spread realistically across different wards or rooms.</p></li>
<li><p>Microorganism Behavior Models: The spread and behaviour of pathogens are modelled based on epidemiological principles, enabling realistic infection dynamics.</p></li>
</ul>
</section>
<section id="model-validation-and-utility" class="level3" data-number="6.10.2">
<h3 data-number="6.10.2" class="anchored" data-anchor-id="model-validation-and-utility"><span class="header-section-number">6.10.2</span> Model Validation and Utility:</h3>
<p>The model’s accuracy was assessed through several methods:</p>
<ul>
<li><p>Micro and Macro-Face Validation: Ensuring that the model aligns with both fine-grained (micro) and broader-scale (macro) expectations.</p></li>
<li><p>Parameter Calibration: Adjusting model parameters based on literature to improve alignment with known infection dynamics.</p></li>
<li><p>Sensitivity Analysis with Expert Input: Working with experts to refine the model and assess its reliability across various scenarios.</p></li>
</ul>
<p>This simulation approach enables hospitals to monitor infection spread, detect patterns over time and space, and use predictive data to make more informed decisions about infection control policies. By addressing data gaps with simulated data, this model compensates for the limited high-volume clinical datasets currently available for AI-driven infection control solutions.</p>


<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list" style="display: none">
<div id="ref-johnson2021" class="csl-entry" role="listitem">
Johnson, Stuart, Valéry Lavergne, Andrew M Skinner, Anne J Gonzales-Luna, Kevin W Garey, Ciaran P Kelly, and Mark H Wilcox. 2021. <span>“Clinical Practice Guideline by the Infectious Diseases Society of America (IDSA) and Society for Healthcare Epidemiology of America (SHEA): 2021 Focused Update Guidelines on Management of Clostridioides Difficile Infection in Adults.”</span> <em>Clinical Infectious Diseases</em> 73 (5): e1029–44. <a href="https://doi.org/10.1093/cid/ciab549">https://doi.org/10.1093/cid/ciab549</a>.
</div>
<div id="ref-sholeh2020" class="csl-entry" role="listitem">
Sholeh, Mohammad, Marcela Krutova, Mehdi Forouzesh, Sergey Mironov, Nourkhoda Sadeghifard, Leila Molaeipour, Abbas Maleki, and Ebrahim Kouhsari. 2020. <span>“Antimicrobial Resistance in Clostridioides (Clostridium) Difficile Derived from Humans: A Systematic Review and Meta-Analysis.”</span> <em>Antimicrobial Resistance &amp; Infection Control</em> 9 (1): 158. <a href="https://doi.org/10.1186/s13756-020-00815-5">https://doi.org/10.1186/s13756-020-00815-5</a>.
</div>
</div>
</section>
</section>

</main> <!-- /main -->
<footer style="text-align: center; margin-top: 20px; padding: 10px; background-color: #f8f9fa;">

  <p>© 2024 Kah Yen Claire Yeak | All rights reserved</p>

</footer>

<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./chapter_4.html" class="pagination-link" aria-label="Diagnostic Techniques and Challenges">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Diagnostic Techniques and Challenges</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./chapter_6.html" class="pagination-link" aria-label="Host immune response">
        <span class="nav-page-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Host immune response</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->




</body></html>